stm32-modules/common/src/nxft15xv103fa2b030.csv

```diff
@@ -0,0 +1,171 @@
+              R25          10  k ohm +/- 1%
```

Comment:
this is an incredibly cursed format my lord

Reply:
🙃